### DISCUSSION TITLE
Copilot won't activate on Windows 10

### DISCUSSION THREAD

**Discussion by tsrdatatech** [community/community#9368] [Copilot Conversations] [answered]
For some reason I cannot get copilot to activate on my windows work computer. It works fine on my MBP. I also installed it for use with PyCharm Community Ed. and it works just fine for that on the same Windows machine. I do not think I get a prompt or something to activate it on Code in Windows.

The output I see is:

[INFO] [default] [2021-12-28T19:59:27.491Z] Experimentation service initialized
[ERROR] [default] [2021-12-28T19:59:27.657Z] GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"
[INFO] [default] [2021-12-28T19:59:27.677Z] Stringified features: {"copilotaa":"false","copilotincludeprompt":"medium"}
[INFO] [default] [2021-12-28T19:59:27.677Z] Feature for copilotaa: false
[INFO] [default] [2021-12-28T19:59:27.677Z] Feature for copilotincludeprompt: medium
[INFO] [default] [2021-12-28T19:59:27.677Z] Feature for copilotset1: undefined
[INFO] [default] [2021-12-28T19:59:27.677Z] Feature for copilotghgh: false
[INFO] [default] [2021-12-28T19:59:27.677Z] Feature for copilotfim: false

Any suggestions for resolving this? 

Btw, I just recently started using Windows after a 15 year absence so I could be missing something.

**Comment by davecheney** [comment, marked as answer]
Hello, I'm sorry you are having difficulties. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by tsrdatatech** [reply]
How is this an answer?

**Comment by tsrdatatech** [comment]
Hi. Yes I am. I was reading about others having a similar issue so I filed a ticket with IT where I work. While this was ongoing we also use PyCharm at my work so I decided to see if copilot worked with that. After installing it I was able to allow access and use it. It seems to work fine in PyCharm as opposed to code so I assume if it works in PyCharm then there must not be any blocking from firewall, vpn etc going on. I am not sure if the extension for code prompts for access or something but that is what happened in PyCharm which allowed me to use copilot so I am thinking perhaps it's an issue with code not prompting me to accept the cert or something? I have a Mac and code and copilot work fine on that as well.

**Comment by renatodru** [comment]
The problem is about trust in chain certificate.
The solution is:
1 - install the extension in VSC "win-ca"
2 - change config "Win-ca: inject (Method to provide fetched root certificates to other externsions)" to **append**.
3 - restart VSC.
